Sources/TuistServer/Utilities/DateService.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+import Mockable
+
+@Mockable
+protocol DateServicing: Sendable {
+    func now() -> Date
+}
+
+struct DateService: DateServicing {
+    func now() -> Date {
+        Date()
+    }
+}
```

Comment:
Can't this be a typealias `() -> Date`, which we can then expose via a `@TaskLocal`?

```swift
extension Date {
  @TaskLocal var now: () -> Date = { Date() }
}
```

Reply:
It can 👍